src/sfProject.ts

```diff
@@ -656,6 +727,34 @@ export class SfProject {
 
     return this.projectConfig;
   }
+
+  public async hasPackageAliases(): Promise<boolean> {
+    return await this.getSfProjectJson().hasPackageAliases();
```

Comment:
```suggestion
    return this.getSfProjectJson().hasPackageAliases();
```
based on: https://jakearchibald.com/2017/await-vs-return-vs-return-await/